iphone/CoreApi/CoreApi/Bookmarks/MWMBookmarksManager.mm

```diff
@@ -667,6 +667,21 @@ - (void)updateBookmark:(MWMMarkID)bookmarkId
     bookmark->SetCustomName(title.UTF8String);
 }
 
+- (void)updateBookmark:(MWMMarkID)bookmarkId
+              setColor:(MWMBookmarkColor)color {
```

Comment:
```suggestion
- (void)updateBookmark:(MWMMarkID)bookmarkId setColor:(MWMBookmarkColor)color {
```
nit: Prefer fewer lines of code if possible (if it fits 120-130 chars), it helps to review code faster.